testing/src/main/java/io/stargate/it/http/docsapi/BaseDocumentApiV2Test.java

```diff
@@ -561,7 +562,7 @@ public void testPutReplacingArrayElement() throws IOException {
     assertThat(OBJECT_MAPPER.readTree(r)).isEqualTo(wrapResponse(fullObjNode, "1", null));
   }
 
-  @Test
+  @RepeatedTest(1000) // TODO: remove repeats once CI passes
```

Comment:
This change is to be reverted before merging.

Reply:
Looks like CI passed. Good to revert?

Reply:
Nice :) reverted